stdlib/2.7/logging/__init__.pyi

```diff
@@ -4,14 +4,14 @@
 
 from typing import Any, Dict, Optional, Sequence, Tuple, overload, Union
 
-CRITICAL = 0
-FATAL = 0
-ERROR = 0
-WARNING = 0
-WARN = 0
-INFO = 0
-DEBUG = 0
-NOTSET = 0
+CRITICAL = 0 # type: int
+FATAL = 0 # type: int
+ERROR = 0 # type: int
+WARNING = 0 # type: int
+WARN = 0 # type: int
+INFO = 0 # type: int
+DEBUG = 0 # type: int
+NOTSET = 0 # type: int
```

Comment:
the extra "type: int" is unnecessary, it follows from the fact that they have value "0" (as opposed to "...")